<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_new_Campaign</class>
 <widget class="QDialog" name="Dialog_new_Campaign">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>162</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New campaign</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_campaign_name">
         <property name="text">
          <string>Campaign &amp;name:</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_campaign_name</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_campaign_name">
         <property name="toolTip">
          <string>Enter name of the new camapign (max. 15 characters).</string>
         </property>
         <property name="maxLength">
          <number>15</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_output_directory">
         <property name="text">
          <string>Output &amp;directory:</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_output_directory</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_output_directory">
         <property name="toolTip">
          <string>Enter the path to the output directory or use the dialog below.</string>
         </property>
         <property name="maxLength">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="pushButton_change_output_directory">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Open dialog to select the output directory.</string>
         </property>
         <property name="text">
          <string>&amp;Change output directory...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
       <property name="centerButtons">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog_new_Campaign</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog_new_Campaign</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
